<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyCompany</class>
 <widget class="QMainWindow" name="MyCompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>工厂管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>欢迎来到管理页面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>添加信息</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAddEmployee"/>
    <addaction name="actionAddCommoditytable"/>
    <addaction name="actionAddDepartment"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>更新信息</string>
    </property>
    <addaction name="actionUpdateEmployee"/>
    <addaction name="actionUpdateCommodity"/>
    <addaction name="actionUpdateDepartment"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查询信息</string>
    </property>
    <addaction name="actionQueryEmployee"/>
    <addaction name="actionQueryCommodity"/>
    <addaction name="actionQureyDepartment"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>删除信息</string>
    </property>
    <addaction name="actionDeleteEmployee"/>
    <addaction name="actionDeleteDepartment"/>
    <addaction name="actionDeleteCommodity"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>商品管理</string>
    </property>
    <addaction name="actionAddCommodity"/>
    <addaction name="actionSaleCommodity"/>
    <addaction name="actionTransferCommodity"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>密码更改</string>
    </property>
    <addaction name="actionUpdatePassword"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddEmployee">
   <property name="text">
    <string>AddEmployee</string>
   </property>
  </action>
  <action name="actionAddCommoditytable">
   <property name="text">
    <string>AddCommoditytable</string>
   </property>
  </action>
  <action name="actionAddDepartment">
   <property name="text">
    <string>AddDepartment</string>
   </property>
  </action>
  <action name="actionaddRecoder">
   <property name="text">
    <string>addRecoder</string>
   </property>
  </action>
  <action name="actionUpdateEmployee">
   <property name="text">
    <string>UpdateEmployee</string>
   </property>
  </action>
  <action name="actionUpdateCommodity">
   <property name="text">
    <string>UpdateCommodity</string>
   </property>
  </action>
  <action name="actionUpdateDepartment">
   <property name="text">
    <string>UpdateDepartment</string>
   </property>
  </action>
  <action name="actionQueryEmployee">
   <property name="text">
    <string>QueryEmployee</string>
   </property>
  </action>
  <action name="actionQueryCommodity">
   <property name="text">
    <string>QueryCommodity</string>
   </property>
  </action>
  <action name="actionQureyDepartment">
   <property name="text">
    <string>QureyDepartment</string>
   </property>
  </action>
  <action name="actionDeleteEmployee">
   <property name="text">
    <string>DeleteEmployee</string>
   </property>
  </action>
  <action name="actionDeleteDepartment">
   <property name="text">
    <string>DeleteDepartment</string>
   </property>
  </action>
  <action name="actionDeleteCommodity">
   <property name="text">
    <string>DeleteCommodity</string>
   </property>
  </action>
  <action name="actionAddCommodity">
   <property name="text">
    <string>AddCommodity</string>
   </property>
  </action>
  <action name="actionSaleCommodity">
   <property name="text">
    <string>SaleCommodity</string>
   </property>
  </action>
  <action name="actionTransferCommodity">
   <property name="text">
    <string>TransferCommodity</string>
   </property>
  </action>
  <action name="actionUpdatePassword">
   <property name="text">
    <string>UpdatePassword</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyCompany.qrc"/>
 </resources>
 <connections/>
</ui>
